src/hooks/useViewport/useViewport.md

```diff
@@ -0,0 +1,45 @@
+### Summary
+
+`useViewport()` can be used to programatically read the viewport size on a React compnent.
+
+It can be useful for conditional rendering, which is usually prefered over hiding with CSS given it reduces the VDOM tree size.
```

Comment:
typo
```suggestion
It can be useful for conditional rendering, which is usually preferred over hiding with CSS given it reduces the VDOM tree size.
```